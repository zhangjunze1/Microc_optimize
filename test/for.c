void main() {
  int n=5;
  int i;
  for (i = 0; i < n; i=i+1)
  {
    print("%d",i);
  }
}
