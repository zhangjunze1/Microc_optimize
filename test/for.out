24 19 0 5 25 13 0 5 12 15 1 13 0 1 1 0 0 12 15 -1 16 45 13 0 1 1 11 22 15 -1 13 0 1 1 13 0 1 1 11 0 1 1 12 15 -1 13 0 1 1 11 13 11 7 18 22 21 1